pkg/apis/core/v1alpha1/storagecluster.go

```diff
@@ -323,6 +323,14 @@ type StorageClusterStatus struct {
 	CollisionCount *int32 `json:"collisionCount,omitempty"`
 	// Conditions describes the current conditions of the cluster
 	Conditions []ClusterCondition `json:"conditions,omitempty"`
+	// Storage represents cluster storage details
+	Storage Storage
+}
+
+// Storage represents cluster storage details
+type Storage struct {
+	// StorageInstancesPerZone describes the amount of instances per zone
+	StorageInstancesPerZone int `json:"storageInstancesPerZone,omitempty"`
```

Comment:
requires too much changes

Reply:
We should not keep it open-ended from our side. It should be a one time change wherever you do. Also for consistency can we rename at least the json tag to `storageNodesPerZone` because we have fields like `maxStorageNodesPerZone` and `maxStorageNodes`.

Reply:
is there a difference between StorageInstancesPerZone and maxStorageNodesPerZone ?

Reply:
if I change int to int32 here, I have to modify `	InstancesPerZone int `json:"instances_per_zone" yaml:"instances_per_zone"`
` in `vendor/github.com/libopenstorage/cloudops/cloud_storage_management.go` as well

Reply:
I don't have to, nvm